### PR TITLE
Update Nano Colors by triggering new build

### PR DESCRIPTION
**Pull request by ai**

### CODE REVIEW

**Comment by ai** [comment]
```
nanocolors     30,585,505 ops/sec
colorette      13,611,611 ops/sec
nanocolors     30,732,318 ops/sec
colorette      13,947,272 ops/sec
nanocolors     29,983,145 ops/sec
colorette      12,966,698 ops/sec
```

**Comment by jorgebucaran** [comment]
It appears to could have broken color functions:

```
> blue()
Uncaught TypeError: Cannot read properties of undefined (reading 'length')
    at /Users/jb/Code/projects/colorette/index.cjs:53:10
> blue(null)
Uncaught TypeError: Cannot read properties of null (reading 'length')
    at /Users/jb/Code/projects/colorette/index.cjs:53:10
> blue(undefined)
Uncaught TypeError: Cannot read properties of undefined (reading 'length')
    at /Users/jb/Code/projects/colorette/index.cjs:53:10
> blue(NaN)
'NaN'
> blue(0)
'0'
> blue(new Date)
'Sun Sep 26 2021 00:46:50 GMT+0900 (Japan Standard Time)'
```

**Comment by ai** [comment]
Thanks. Fixed in 0.2.5.

**Comment by jorgebucaran** [comment]
Not quite yet. These should produce color output too, but as you can see they currently do not.

```
> blue(0)
'0'
> blue(1)
'1'
> blue(NaN)
'NaN'
> blue(null)
'null'
> blue(Infinity)
'Infinity'
> blue(true)
'true'
> blue(false)
'false'
> blue(new Date)
'Sun Sep 26 2021 01:21:34 GMT+0900 (Japan Standard Time)'
```

**Comment by ai** [comment]
It is expected trade-off since anyway it is against types (so out of public API).

**Comment by jorgebucaran** [comment]
These should not print `undefined`, but `""`.
```
> blue()
'undefined'
> blue(undefined)
'undefined'
```

**Comment by ai** [comment]
I prefer behavior with `"undefined"`. Anyway, it is out of public API and any behavior is possible.

**Comment by jorgebucaran** [comment]
> It is expected trade-off since anyway it is against types (so out of public API).

Now you'll have to explicitly cast numbers and dates, which are not uncommon. That's a bug. Printing `undefined` is odd too.

If you are not going to fix that, then how could this benchmark be accurate? Chalk, kleur, etc., all properly colorize numbers, and dates, and `null`. 

nanocolors currently gets significantly more op/s because you are not handling several edge cases. Sounds irresponsible to me, not to mention you introduced a breaking change to your [13K+](https://github.com/ai/nanocolors/network/dependents?package_id=UGFja2FnZS0yNzI0MDQ2MTkx) dependants.

**Comment by ai** [comment]
OK. Fixed in 0.2.6.

**Comment by jorgebucaran** [comment]
Nice, but now `blue()`, etc., print `undefined` in blue, which is definitely a bug. 😂

**Comment by ai** [comment]
> Nice, but now blue(), etc., print undefined in blue, which is definitely a bug. joy

It is not a bug. I explicitly defined public API, and it doesn’t violate it.

If you do not want to merge PR, just say it directly.

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "license": "MIT",
   "devDependencies": {
     "benchmark": "^2.1.4",
-    "nanocolors": "*",
+    "nanocolors": ">=0.2.6",
```
Comment (jorgebucaran) [inline package.json:6]: @ai If you leave `*` it will grab the latest version, right?

Comment (ai) [reply package.json:6]: Yeap. Version as a string can be used to trigger results update later.

**Comment by jorgebucaran** [comment]
> It is not a bug. I explicitly defined public API, and it doesn’t violate it.

IMO you should consider supporting it, but I get your point.

**Comment by jorgebucaran** [comment]
👌

**Comment by jorgebucaran** [comment]
@ai Here are the results:

```console
nanocolors     25,231,475 ops/sec
colorette      24,805,912 ops/sec
nanocolors     24,238,338 ops/sec
colorette      24,990,408 ops/sec
nanocolors     24,789,703 ops/sec
colorette      25,016,693 ops/sec
```

**Comment by ai** [comment]
This result is after you backported my optimization. Hope you will mention me in `colorette` docs.

**Comment by jorgebucaran** [comment]
Replied here: https://github.com/babel/babel/pull/13783#issuecomment-927258580.

**Comment by jorgebucaran** [comment]
Locking to avoid comment edits. Just in case, one never knows.

**Pull request merged**